shellai/config.py

```diff
@@ -0,0 +1,150 @@
+import json
+import logging
+import sys
+from collections import namedtuple
+from pathlib import Path
+
+if sys.version_info >= (3, 11):
+    import tomllib
+else:
+    import tomli as tomllib
```

Comment:
I generally prefer feature checks instead of version checks:
```suggestion
try:
    # For Python-3.11 and greater
    import tomllib
except ImportError:
    import tomli as tomllib
```

Reply:
done locally